src/server/menu-controller.js

```diff
@@ -86,6 +88,10 @@ class MenuController {
       const menu = await this.menuService.saveMenu({ menu: req.body, user: req.user });
       return res.send({ menu });
     });
+    router.delete('/api/v1/menus', [needsPermission(permissions.MIGRATE_DATA), jsonParser], async (req, res) => {
```

Comment:
No need for JSON parser here (although it doesn't really screw it up) :-)

Reply:
What parser? 🙄 😝